staging/src/k8s.io/apiserver/pkg/admission/metrics/metrics.go

```diff
@@ -45,8 +45,8 @@ const (
 )
 
 var (
-	// Use buckets ranging from 5 ms to 2.5 seconds (admission webhooks timeout at 30 seconds by default).
-	latencyBuckets       = []float64{0.005, 0.025, 0.1, 0.5, 2.5}
+	// Use buckets ranging from 5 ms to 10 seconds (admission webhooks timeout at 30 seconds by default).
+	latencyBuckets       = []float64{0.005, 0.025, 0.1, 0.5, 2.5, 5.0, 10.0}
```

Comment:
hm, I'd expect a bucket right at 1s (that's the longest I think a well behaved webhook should be taking). The 5.0 bucket doesn't follow the pattern of the past increases, which are x5 or x4 each time. It might be useful to know the difference between 10s (terrible performance) and 20-29s (absolutely unacceptable). I don't think this is blocking, feel free to make a follow-up.